zha/application/platforms/sensor/__init__.py

```diff
@@ -752,6 +774,27 @@ class ElectricalMeasurementRMSVoltage(PolledElectricalMeasurement):
     _div_mul_prefix = "ac_voltage"
 
 
+@MULTI_MATCH(cluster_handler_names=CLUSTER_HANDLER_ELECTRICAL_MEASUREMENT)
+class ElectricalMeasurementRMSVoltagePhB(ElectricalMeasurementRMSVoltage):
+    """RMS voltage phase B measurement."""
+
+    _attribute_name = "rms_voltage_ph_b"
+    _unique_id_suffix = "rms_voltage_ph_b"
+    _attr_translation_key: str = "rms_voltage_ph_b"
+    _skip_creation_if_no_attr_cache = True
+    _attr_max_attribute_name = "rms_voltage_max_ph_b"
+
+
+@MULTI_MATCH(cluster_handler_names=CLUSTER_HANDLER_ELECTRICAL_MEASUREMENT)
+class ElectricalMeasurementRMSVoltagePhC(ElectricalMeasurementRMSVoltagePhB):
```

Comment:
Let's remove the inheritance from `ElectricalMeasurementRMSVoltagePhB` and change it to `ElectricalMeasurementRMSVoltage`, similar to the other PR.
`_skip_creation_if_no_attr_cache = True` has to be added then.

Reply:
Sorry, I intended to change that and forgot to do it